util/test/predictoor/test_models.py

```diff
@@ -0,0 +1,62 @@
+import pytest
```

Comment:
We already discussed this. The directory structure needs to change. This test & siblings need to move from `util/test/predictoor/test_*.py` to `util/predictoor/test_*.py`. It's more maintainable to have tests at leaf nodes, versus replicate a hierarchy of directories.

Reply:
Yep sorry, I didn't see Trent's comment before commenting the same thing a few lines above. But Berkay also didn't know about this when he started the PR :)

Reply:
Yes